includes/payment-tokens/class-wc-stripe-payment-tokens.php

```
@@ -552,13 +552,39 @@ private function add_token_to_user( $payment_method, WC_Stripe_Customer $custome
 
 		$found_token = $this->get_duplicate_token( $payment_method, $customer->get_user_id(), $gateway_id );
 		if ( $found_token ) {
+			$token_updated = false;
+
 			// Update the token with the new payment method ID if the current payment method ID is not in the list of payment method IDs retrieved from Stripe.
 			if ( ! in_array( $found_token->get_token(), $payment_method_ids, true ) ) {
```

PHPMD (2.15.0)

[error] The variable $payment_method_ids is not named in camelCase. (undefined)

(CamelCaseVariableName)